Analytics Engineer/code/src/assets/scripts/google-analytics.js

```diff
@@ -0,0 +1,4 @@
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);}
+gtag('js', new Date());
+gtag('config', 'UA-43652404-4');
```

Comment:
Same thing here, if the ID being exposed is a concern, environmental variables should be used.